app/helpers/camaleon_cms/frontend/application_helper.rb

```diff
@@ -21,16 +21,16 @@ def cama_url_to_fixed(url_to, *args)
     options = args.extract_options!
     _current_site = options.delete(:site) || current_site
     if request.present?
-      if options.include?(:locale) && options[:locale] == false
+      if options[:locale] == false
         options.delete(:locale)
       else
-        options[:locale] = I18n.locale if !options[:locale].present? && _current_site && _current_site.get_languages.size > 1
+        options[:locale] = I18n.locale if options[:locale].blank? && _current_site&.get_languages.size > 1
```

Comment:
This refactoring is not equivalent. If `_current_site` is nil, then the `size` method will be called on `nil` and return a `NoMethodError`.

Reply:
Only `get_languages` won't be called. The safe navigation operator doesn't end the chain, it continues the chain with nil as the receiver of the next method.

```ruby
nil&.get_languages.size

# `<main>': undefined method `size' for nil:NilClass (NoMethodError)
```

It's equivalent to this:

```ruby
(_current_site.nil? ? nil : _current_site.get_languages).size
```

That's why RuboCop complains if you use safe navigation then continue the chain with regular dots instead of more safe navigation. The safe thing would be to put it back how it was, or do this:
```ruby
_current_site&.get_languages&.size&.> 1
```

Reply:
Finally I have tried this in console and now I am understanding it!

Thanks, @brian-kephart !